### ISSUE TITLE
[Suggestion] Confirm/Back depending on PS3 O/X button assignment

### ISSUE THREAD

**Issue by sheltpat** [bucanero/apollo-ps3#117]
Hey there,

I want to suggest an improvement to our beloved Apollo, which is to assign O/X to Confirm/Back following the system assignment.
I'm using my PS3 with swapped O/X functionality and force myself to remember that Apollo uses the opposite (default actually). This would make it behave in line with all other applications I use.

**Actual behavior:**
Apollo considers X to confirm, O to cancel (possibly due to hardcoding?), even on swapped O/X system setting.

**Desired behavior:**
Follow system assignment if user has set their PS3 to O to confirm, X to cancel.

I don't use Apollo every day, but this would make it much quicker for me and possibly a few others, as I would not have to think twice before pressing a button. ;)

Thanks for your continued dedication and best wishes.

**Comment by bucanero** [comment]
Yes, this could be added , as I've already done this code to detect the current system setting for the "OK" button (cross or circle).

I'm focusing on other projects so I can't really tell when I'll be back on PS3 stuff, but I'll keep it in mind.

**Comment by bucanero** [comment]
correctly fixed in 27ee1021f670fa915e6ff629a43f72eaf898fd8c
